examples/lock-app/efr32/src/AppTask.cpp

```diff
@@ -127,6 +202,8 @@ void AppTask::AppTaskMain(void * pvParameter)
     }
 
     EFR32_LOG("App Task started");
+    chip::DeviceLayer::ConnectivityMgr().AddCHIPoBLEConnectionHandler(&AppTask::HandleBLEConnectionOpened);
+    SetDeviceName("LockDemo._chip._udp.local.");
```

Comment:
nit: 
```suggestion
    SetDeviceName("EFR32LockDemo._chip._udp.local.");
```
Since there is another nRF5 lock demo.